src/main/java/seedu/address/logic/commands/quiz/NextQuestionCommand.java

```diff
@@ -5,23 +5,30 @@
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.model.Model;
+import seedu.address.model.quiz.Quiz;
+import seedu.address.model.quiz.QuizManager;
 
 /**
  * Command to go to the next question
  */
 public class NextQuestionCommand extends Command {
     public static final String COMMAND_WORD = "next";
-
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Quizes the next question, takes no arguments";
+    public static final String MESSAGE_SUCCESS = "Got Next Question!";
+
+    private final QuizManager quizManager;
 
-    public static final String MESSAGE_SUCCESS = "Got Next Quiz!";
-    public NextQuestionCommand() {}
+    public NextQuestionCommand(QuizManager quizManager) {
+        this.quizManager = quizManager;
+    }
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        // TODO: use the QuizManager.nextQuestion() method
-        return new CommandResult(MESSAGE_SUCCESS);
+        quizManager.nextQuestion();
```

Comment:
I think we need to catch the QuizOutOfBoundException thrown by nextQuestion() somewhere